pages/web-lit-basics.jsx

```diff
@@ -44,7 +44,7 @@ var curriculumList = [
         image1x: "/img/pages/web-lit-basics/img-kraken-code.jpg",
         image2x: "/img/pages/web-lit-basics/img-kraken-code@2x.jpg",
         subtitle: "Understanding credibility",
-        description: "Learners use the Internet to solve the mystery of The Kraken, a legendary sea creature, while also learning about search terms, keywords, and how to assess the validity and relevance of web sources.",
+        description: "Learners use the Internet to solve the mystery of The Kraken, a legendary sea creature, while also learning about search terms, keywords, and how to assess the validity and relevance of web sources. This activity was made by <a href='http://mouse.org'>MOUSE</a>.",
```

Comment:
the styling is controlled by https://github.com/mozilla/teach.mozilla.org/blob/develop/less/pages/web-lit-basics.less, and we need to reevaluate our choice of https://github.com/mozilla/teach.mozilla.org/blob/develop/less/pages/web-lit-basics.less#L11, which turns of any and all link decorations. It's definitely not accessible, and since we don't even colour the links differently, impossible to tell there's a link even for "normal" users =x